### ISSUE TITLE
ds000214 Failed MRIQC

### ISSUE THREAD

**Issue by sadevp** [poldracklab/mriqc#642]
Hello,

Dataset: ds000214 failed after running MRIQC. It can be found at this link: https://openneuro.org/datasets/ds001060

Thanks.

**Comment by oesteban** [comment]
Fixed link: https://openneuro.org/datasets/ds001060/versions/00001

**Comment by oesteban** [comment]
Hi @sadevp,

The logs are saying the following:

```
Traceback (most recent call last):
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/pipeline/plugins/base.py", line 255, in run
  File "/usr/local/miniconda/lib/python3.6/site-packages/mriqc/bin/mriqc_run.py", line 325, in main
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/pipeline/plugins/base.py", line 308, in _clean_queue
    result=result)
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/pipeline/plugins/multiproc.py", line 190, in _report_crash
    traceback=result['traceback'])
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/pipeline/plugins/base.py", line 78, in report_crash
    crash2txt(crashfile, dict(node=node, traceback=traceback))
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/utils/filemanip.py", line 596, in crash2txt
    fp.write(''.join(record['traceback']))
OSError: [Errno 28] No space left on device
```

I will ask @chrisfilo the best way to report this in OpenNeuro.

Thanks for reporting!

**Comment by chrisfilo** [comment]
Good call. https://github.com/OpenNeuroOrg/openneuro/issues

**Comment by oesteban** [comment]
Done: https://github.com/OpenNeuroOrg/openneuro/issues/55

**Comment by sadevp** [comment]
I should continue reporting any failed MRIQC datasets here<https://github.com/poldracklab/mriqc/issues>, right?

Thanks for your help!

- Sadev



On Sep 11, 2017, at 10:31 PM, Oscar Esteban <notifications@github.com<mailto:notifications@github.com>> wrote:


Closed #642<https://github.com/poldracklab/mriqc/issues/642>.

**Comment by oesteban** [comment]
Yes, I don't mind triaging them and propagate the issue to other places if necessary.
